region/tourian/main.json

```diff
@@ -3737,54 +3813,6 @@
           "nodeSubType": "closedWall",
           "nodeAddress": "0x001aae0",
           "doorEnvironments": [{"physics": "air"}],
-          "runways": [
-            {
-              "name": "Base Runway - Tourian Escape Room 1 Right Door (to Mother Brain)",
-              "length": 2,
-              "strats": [
-                {
-                  "name": "Base",
-                  "notable": false,
-                  "requires": []
-                }
-              ],
-              "openEnd": 0
-            },
-            {
-              "name": "Diagonal Shot Runway - Tourian Escape Room 1 Right Door (to Mother Brain)",
```

Comment:
These runways would still be used for `canLeaveShinecharged` out the bottom door.

Reply:
I see now that shinecharges are meant for a separate PR

Reply:
I was actually debating about this and took it out. I felt like getting through was harder than getting the 21 tile short charge. But that might not be the case with some beams. I'll look into it more

Reply:
That may be the appropriate way to look at it.  It is the same way of doing the room as speedruns, but they would have hyper beam.  And even then I wouldn't expect them to know the movements until `VH`